documentation/source/building.rst

```diff
@@ -33,6 +33,10 @@ Make Variables
     ``SIM``
       Selects which simulator Makefile to use.  Attempts to include a simulator specific makefile from :file:`cocotb/share/makefiles/makefile.$(SIM)`
 
+    ``WAVES``
+      Set this to 1 to enable wave traces dump for the Aldec and Questa simulators.
```

Comment:
I guess we should really say "for the Aldec Riviera-PRO and Mentor Graphics Questa simulators."

Reply:
I changed name according to your review.